Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -831,7 +844,7 @@ class HTTPClientTests: XCTestCase {
             XCTAssertNoThrow(try web.writeOutbound(.head(.init(version: .init(major: 1, minor: 0),
                                                                status: .ok,
                                                                headers: HTTPHeaders([("connection",
-                                                                                      i % 2 == 0 ? "close" : "keep-alive")])))))
+                                                                                     i % 2 == 0 ? "close" : "keep-alive")])))))
```

Comment:
Mind removing this extraneous change?

Reply:
no prob -- I'll check asap (basically I'm using swiftlint)

Reply:
i checked & I've use the` swiftformat --swiftversion 5 .` before push